daemon/mgr/container.go

```diff
@@ -999,16 +1006,12 @@ func (mgr *ContainerManager) Update(ctx context.Context, name string, config *ty
 		}
 	}
 
-	// Update Env
-	newEnvSlice, err := mergeEnvSlice(config.Env, c.Config.Env)
-	if err != nil {
-		return err
-	}
-	c.Config.Env = newEnvSlice
-
 	if mgr.containerPlugin != nil {
-		if err = mgr.containerPlugin.PostUpdate(c.BaseFS, c.Config.Env); err != nil {
-			return err
+		// post update env file
+		if len(config.Env) > 0 {
```

Comment:
how about `if len(config.Env) > 0 && err := mgr.containerPlugin.PostUpdate(c.BaseFS, c.Config.Env); err != nil` ?

Reply:
Too long, what do you think?

Reply:
Look like conbine `mgr.containerPlugin != nil && len(config.Env) > 0` better?

Reply:
can not agree no more

Reply:
Updated.